packages/sdk-codegen-scripts/src/utils.ts

```diff
@@ -65,10 +65,29 @@ export interface IGenProps {
   apis: string[]
   /** Last API version */
   lastApi: string
+  /** Skip generating streams files? */
+  noStreams: boolean
 }
 
 const generatorHelp = () => {
-  log(`sdkGen [languages...] [-v|--versions <versions file>] [-h|--help]`)
+  log(
+    `sdkGen [languages...] [-v|--versions <versions file>] [-n|--nostreams] [-h|--help]
+  languages...:   zero or more language specifiers separated by space or comma. Defaults to all supported languages.
+  -v|--versions:  location of a JSON versions file in ILookerVersions format to read for getting specs
+  -n|--nostreams: skip generation of a language SDK 'streams' files (if it supports streaming)
+  -h|--help:      display this output
+
+  examples:
+    # Generates all supported SDKs
+    yarn gen
+
+    # Generates Typescript and Python SDKs
+    yarn gen ts,py
```

Comment:
so does it accept the file extension? I've been using `yarn gen Typescript`

Reply:
yup. extension, label, or language name matches. Added that a little while ago.

Reply:
because I'm lazy